### PR TITLE
fix(header): Header fixups

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

**Comment by jefflembeck** [comment]
Seeing this from 652 -> 767px

pivotal ui light header 2016-05-25 14-18-08

File: src/pivotal-ui/components/header/header.scss [hunk 1]
```diff
@@ -89,7 +89,6 @@ header > .header-item {
 }
 header > .brand-container {
   order: 1;
-  width: 50%;
```
Comment (jefflembeck) [inline src/pivotal-ui/components/header/header.scss:92]: Why remove this?

Comment (aredridel) [inline src/pivotal-ui/components/header/header.scss:92]: Oh hey. That causes the above bug!

:P 

Yay change blindness!

**Comment by aredridel** [comment]
Should be fixed!

File: src/pivotal-ui/components/header/package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "homepage": "http://styleguide.pivotal.io/objects.html#header",
   "dependencies": {
-    "@npmcorp/pui-css-drop-down-menu": "^1.3.4-0",
+    "@npmcorp/pui-css-drop-down-menu": "^1.3.4",
     "@npmcorp/pui-css-typography": "^4.0.0"
   },
-  "version": "3.0.0"
+  "version": "3.0.1-1"
```
Comment (nexdrew) [inline src/pivotal-ui/components/header/package.json]: Not that it really matters, but should this have been `3.0.1-alpha.1` instead? If you used a tool to automatically bump this version, then please ignore this.

Comment (aredridel) [inline src/pivotal-ui/components/header/package.json]: I've never used `-alpha.` but could start. I don't think it adds much info though. It works the same from a semver point of view.

Comment (nexdrew) [inline src/pivotal-ui/components/header/package.json]: Ok, that was mainly for my own benefit. Hopefully we can start using tools to bump versions for us so it's no longer an issue.

**Comment by jefflembeck** [comment]
This is:

impossibly well documented ❤️ 
looks incredible.

The only thing I have nitpick wise is that tabbing to the search button doesn't indicate anything, active/focus wise, but I think that's not a blocker.

🐑 this. 🐑 it good.

**Pull request merged**